pandas/core/generic.py

```diff
@@ -2525,24 +2525,35 @@ def _take(self, indices, axis=0, convert=True, is_copy=True):
             The axis on which to select elements. "0" means that we are
             selecting rows, "1" means that we are selecting columns, etc.
         convert : bool, default True
-            .. deprecated:: 0.21.0
-               In the future, negative indices will always be converted.
-
             Whether to convert negative indices into positive ones.
             For example, ``-1`` would map to the ``len(axis) - 1``.
             The conversions are similar to the behavior of indexing a
             regular Python list.
+            .. deprecated:: 0.21.0
+               In the future, negative indices will always be converted.
         is_copy : bool, default True
             Whether to return a copy of the original object or not.
+        kwargs : xxx, default xxx
+            TODO MUST ADD A REASONABLE DESCRIPTION.
```

Comment:
optional keyword arguments to ``:meth:`pandas.compat.numpy.function.validate_take`

Reply:
Fix in later commit